components/CredentialCard/CredentialCard.tsx

```diff
@@ -132,13 +133,13 @@ const mapCredDataToDisplayValues = (credential?: VerifiableCredential): Credenti
     }
   }
   const common = {
-    issuedTo: credential.credentialSubject.name || credential.name,
+    issuedTo: credential.credentialSubject?.name ??  credential.name ?? extractNameFromOBV3Identifier(credential.credentialSubject),
```

Comment:
```suggestion
    issuedTo: credential.credentialSubject?.name ??  extractNameFromOBV3Identifier(credential.credentialSubject) ?? credential.name,
```
I think we should shift the 'extract' function to the middle -- the usecase where we're using `credential.name` is a last resort, when the other two options are absent.